### PR TITLE
Add git blob hash to `version` cmd

### PR DESCRIPTION
**Pull request by theguy147**
### Description/Motivation/Screenshots ###

see #682

#### `gef.py` inside git dir

image

### Checklist ###

- [x] My PR was done against the `dev` branch, not `master`.
- [x] My code follows the code style of this project.
- [x] My change includes a change to the documentation, if required.
- [x] My change adds tests as appropriate.
- [x] I have read and agree to the **CONTRIBUTING** document.

### CODE REVIEW

File: gef.py [hunk 1]
```diff
@@ -4323,7 +4323,9 @@ def do_invoke(self, argv):
             extra = "dirty" if len(subprocess.check_output('git ls-files -m', cwd=gef_dir, shell=True).decode("utf8").strip()) else "clean"
```
Comment (Grazfather) [inline gef.py:4323]: nit: Can you fix this line and previous to use double quotes? I believe in fixing nearby lines :)

File: gef.py [hunk 2]
```diff
@@ -4323,7 +4323,9 @@ def do_invoke(self, argv):
...
             gef_print("GEF: rev:{} (Git - {})".format(ver, extra))
         else:
+            gef_blob_hash = subprocess.check_output("git hash-object {}".format(gef_fpath), shell=True).decode().strip()
             gef_print("GEF: (Standalone)")
+            gef_print("Blob Hash({}): {}".format(gef_fpath, gef_blob_hash))
```
Comment (Grazfather) [inline gef.py:4328]: Maybe add to our issue template how to use this?

Comment (theguy147) [reply gef.py:4328]: Usage hasn't changed in my opinion. The Blob Hash is only printed in `standalone` mode and should be included when somebody opens an issue.

Or do you mean a little documentation on how the commit can be found using this blob hash from a developers point of view?

Comment (Grazfather) [reply gef.py:4328]: Yes, I mean even a comment for the issue writer to maybe find the commit using this.

It's not really on them, but I am afraid that I myself will forget how to use this information. Maybe a blurb somewhere on a debugging doc, even.

Or maybe I should just remember how. 👀

Comment (theguy147) [reply gef.py:4328]: ok, i'll just add the `version` command to the documentation then

**Comment by Grazfather** [comment]
Looks great. Thanks a lot.

**Comment by theguy147** [comment]
oh, no! I made a "typo" in the documentation.

**Pull request merged**